_partials/scripts.blade.php

```diff
@@ -1,3 +1,4 @@
+{!! Assets::getJsVars() !!}
```

Comment:
I was thinking of doing this mannually like:
```
@php $defaultCurrency = app('currency')->getDefault(); @endphp
<script>
var window.app.currency = {...}
</script>
```

But then, there's a `theme.php` file designed to handle stuff like this, the feature is partially documented and now complete with these commits https://github.com/tastyigniter/TastyIgniter/commit/dd37688b117a87918e33a4a42d6d5df0f06069ca & https://github.com/tastyigniter/TastyIgniter/commit/5c4b69896e0c913c18cae2577633e800a99f310d. 

So in the theme.php file, add this
```
$defaultCurrency = app('currency')->getDefault();
Assets::putJsVars([
    'currency' => [
            'symbol' => $defaultCurrency->currency_symbol,
            'symbol_position' => $defaultCurrency->symbol_position,
            'thousand_sign' => $defaultCurrency->thousand_sign,
            'decimal_sign' => $defaultCurrency->decimal_sign,
            'decimal_precision' => $defaultCurrency->decimal_position,
        ],
]);
```

See we don't need the `getFrontEndVars` method really.

Reply:
Ok, I think I've done what you intended - let me know if not